src/call/mod.rs

```diff
@@ -541,12 +534,12 @@ impl WriteFlags {
     }
 
     /// Get whether buffer hint is enabled.
-    pub fn get_buffer_hint(&self) -> bool {
+    pub fn get_buffer_hint(self) -> bool {
```

Comment:
This seems to be an API change. Is this intentional? (also the below function)

They seem to now consume the `self`, which seems unnecessary for ` `get_*` check.

Reply:
`self` is `Copy`, so even it's an API change, it's backward compatible.